test/table2qb/pipelines/cube_test.clj

```diff
@@ -45,7 +45,20 @@
         dimension-names ["year" "flow" "code"]]
     (is (= "http://example.com/data/test/{+year}/{+flow}/{+code}" (observation-template domain-data-prefix dataset-slug dimension-names)))))
 
-(deftest read-component-specifications-test
+(defn- example-csvw-schema-part [example-directory part-name]
+  "Extracts a table schema from the json metadata that covers the whole cube"
+  (let [metadata-file (example-csvw example-directory (str example-directory "-metadata.json"))
+        schema (util/read-json metadata-file)
+        part-number (case part-name
+                      "dataset" 0
```

Comment:
Ordinal matching isn't particularly robust, but it's at least simpler than matching by e.g. the aboutURL template.